_topic_map.yml

```diff
@@ -1182,7 +1186,10 @@ Topics:
   File: architecture-component-imageregistry
 - Name: Image Registry Operator in OpenShift Container Platform
   File: configuring-registry-operator
-  Distros: openshift-enterprise,openshift-origin
+  Distros: openshift-enterprise
+- Name: Image Registry Operator in OKD
```

Comment:
@bmcelvee Can you please review this? This is to fix "OpenShift Container Platform" from appearing in the TOC in the OKD docs.

Reply:
LGTM, thank you!